src/main/java/pl/sdacademy/carrental/controllers/BranchController.java

```diff
@@ -21,9 +23,9 @@ public BranchController(final BranchService branchService) {
    
    @GetMapping("/list-branches")
    public String listAllBranches(final Model model) {
-      final Map<Branch, Long> branchesWithCarCount = branchService.getBranchesWithCarCount();
+      final Map<Branch, Map<CarCategory, Long>> branchesWithCarCount = branchService.getBranchesWithCarCount();
```

Comment:
see Table from Guava that wraps Maps na use Map as a value

Reply:
tried it… iterating through the table and getting necessary info seemed like a nightmare